### ISSUE TITLE
Trigger saving of default values

### ISSUE THREAD

**Issue by RathW** [pluginsGLPI/fields#695]
Dear all,
we have installed GLPI 10.0.9 together with the Additional Fields plugin 1.21.1.

AO we are using the additional fields in -> Assets -> software 
When entering new software manually, everything is doing fine and default values were selected and saved.

But if a new software is added by the glpi-agent the default values were not stored.

Only after opening such a new, automatic added software entry and saving it, the default values were selected as expected.

Is there a way to trigger/force the filling of the default values for these automatically generated items?

Thanks in advance!
Kind regards 
Wolfgang

**Comment by stonebuzz** [comment]
There has been no activity on this issue for some time and therefore it is considered stale

If this issue is related to a bug, please try to reproduce on latest release (GLPI and plugin). If the problem persist, feel free to open a new issue.

If it is related to a new feature, please open a topic to discuss with community about this enhancement on [suggestion website](https://glpi.userecho.com/).

You may also consider taking a [subscription](https://glpi-project.org/subscriptions/) to get professionnal support or [contact GLPI editor team](https://portal.glpi-network.com/contact-us) directly.
